src/docfx/lib/HtmlUtility.cs

```diff
@@ -18,6 +18,54 @@ internal static class HtmlUtility
     {
         public delegate void TransformHtmlDelegate(ref HtmlReader reader, ref HtmlWriter writer, ref HtmlToken token);
 
+        public static readonly string[] HtmlMetaHidden = new[]
```

Comment:
This doesn't need to be public.
```suggestion
        private static readonly string[] s_htmlMetaHidden = new[]
```